prody/atomic/atomgroup.py

```diff
@@ -1204,15 +1254,35 @@ def setCSLabels(self, labels):
         else:
             raise TypeError('labels must be a list')
 
-    def setBonds(self, bonds):
+    def setBondOrders(self, bondOrders):
```

Comment:
Please check if bondOrders is None before checking its len, e.g. 

        if bondOrders is None:
            self._bondOrders = bondOrders
            return
        
        if len(bondOrders)!=len(self._bonds):
            raise ValueError('invalid bond order list, bond and bond order length mismatch')
        if min(bondOrders)<1 or max(bondOrders)>5:
            raise ValueError('invalid bond order value, values must range from 1 to 5')

        self._bondOrders = np.array(bondOrders, np.int8)